frontend/src/components/stages/chat_interface.ts

```diff
@@ -222,18 +223,29 @@ export class ChatInterface extends MobxLitElement {
       this.readyToEndDiscussionLoading = false;
     };
 
-    return html`
+    const isDisabled =
+      this.participantService.disableStage ||
+      this.participantService.isReadyToEndChatDiscussion(
+        this.stage.id,
+        currentDiscussionId
+      );
+    const buttonHTML = html`
       <pr-button
         color="tertiary"
         variant="tonal"
-        ?disabled=${this.participantService.disableStage
-          || this.participantService.isReadyToEndChatDiscussion(this.stage.id, currentDiscussionId)}
+        ?disabled=${isDisabled}
         ?loading=${this.readyToEndDiscussionLoading}
         @click=${onClick}
       >
         Ready to end discussion
       </pr-button>
     `;
+    return isDisabled
+      ? html`<pr-tooltip text="You can move on once others are also ready to move on."
```

Comment:
FYI: You can set pr-tooltip text="" if you want the tooltip to not show. So a cleaner version might be to always have the tooltip wrapped around the button, but conditionally render the tooltip text.